tests/e2e/provisioning_e2e/tests/test_sync_symmetric_enrollments.py

```diff
@@ -6,31 +6,28 @@
 
 from provisioning_e2e.service_helper import Helper, connection_string_to_hostname
 from azure.iot.device import ProvisioningDeviceClient
-from azure.iot.device.common import X509
-from provisioningserviceclient import ProvisioningServiceClient, IndividualEnrollment
-from provisioningserviceclient.protocol.models import AttestationMechanism, ReprovisionPolicy
+from ..provisioningservice.protocol.models import (
```

Comment:
Consider just importing `models` and then referring to the specific models through that module to cut down on unnecessary imports. Here and everywhere.